models/repo.go

```diff
@@ -1646,7 +1646,7 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (repos []*Repository, _ int
 		}
 	}
 	if len(opts.Keyword) > 0 {
-		sess.And("repo.lower_name LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%")
+		sess.Where("repo.lower_name LIKE ? OR repo.description LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%", "%"+strings.ToLower(opts.Keyword)+"%")
```

Comment:
Have you tested a logged in user, who is a collaborator of other private repositories?

Because at one time I was troubled by `Where` that it discards previous conditions and stats over. Similiar situation here: https://github.com/gogits/gogs/pull/4548/files#diff-d4e2553a87eb8b329efd7335f028903cR1641